Bio/Align/AlignInfo.py

```diff
@@ -487,7 +486,7 @@ def information_content(
            first position in the seq is 203 in the initial sequence, for
            the info content, we need to use zero). This defaults to the entire
            length of the first sequence.
-         - e_freq_table - A FreqTable object specifying the expected frequencies
+         - e_freq_table - A dictionary specifying the expected frequencies
```

Comment:
Maybe say "A FreqTable object or dictionary ..." here?

Reply:
A FreqTable is a dictionary.